translate data ingest routine from python to ruby rake task

whunter opened the pull request:
rake task functioning with helpers in lib

move driver script into DataIngest class in lib so it can be called from application or rake task. Adjust paths to match system as setup by install_blacklight.sh

Call using:

``` bash
bundle exec rake geoblacklight:data_ingest
```

pmather commented on lib/vtul/dataingest.rb:
```diff
@@ -0,0 +1,184 @@
+  end
+
+
+  def ingestrec(solrurl, content)
```
FWIW, the GeoBlacklight gem has a rake task that includes one to ingest JSON-encoded GeoHydra records into Solr.  (See [this rake task for details](https://github.com/geoblacklight/geoblacklight/blob/master/lib/tasks/geoblacklight.rake#L13).)

The heavy lifting in that rake task is done via a call to Blacklight.default_index.connection.add, which knows how to reach the Solr endpoint (and presumably also handles any other Blacklight side effects we might not know about at this stage).  This might avoid you having to mess around with trying to process the config/blacklight.yml file yourself to figure out where the core is that you want to use, which is what you do later in the run() method.

It may also be that Blacklight.default_index.connection.add might throw exceptions that we may be able to catch to do error checking/logging.  It may we worth looking into...

yinlinchen replied:
Does this function output some kind of logs for us to parse and generate reports for Shane?

ghoster commented:
Please remove scripts/ and its files.

yinlinchen commented:
please remove scripts folder and its files

whunter committed:
switch to asset pipeline handler for image

update from python to ruby

rake task functioning with helpers in lib

move driver script into DataIngest class in lib so it can be called from application or rake task. Adjust paths to match system

refactor to use Blacklight.default_index.connection to connect and push to Solr instead of doing it manually

more accurate method name

change naming of lib directory. Check if directory exists before continuing rake task

log directory does not exist error to file
